apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Snippets/SubscriberClientSnippets.cs

```diff
@@ -219,19 +205,17 @@ public void Pull()
             string subscriptionId = _fixture.CreateSubscriptionId();
 
             PublisherClient publisher = PublisherClient.Create();
-            string topicName = PublisherClient.FormatTopicName(projectId, topicId);
+            TopicName topicName = new TopicName(projectId, topicId);
             publisher.CreateTopic(topicName);
             PubsubMessage newMessage = new PubsubMessage { Data = ByteString.CopyFromUtf8("Simple text") };
-            SubscriberClient.Create().CreateSubscription(SubscriberClient.FormatSubscriptionName(projectId, subscriptionId), topicName, null, 60);
+            SubscriberClient.Create().CreateSubscription(new SubscriptionName(projectId, subscriptionId), topicName, null, 60);
             publisher.Publish(topicName, new[] { newMessage });
 
             // Snippet: PullAsync(string,*,*,CallSettings)
             // Additional: PullAsync(string,*,*,CancellationToken)
```

Comment:
Done - will leave that as a separate commit, to avoid issues with the LongRunning case-only change getting messed up by a rebase.